src/Value.zig

```diff
@@ -691,6 +691,12 @@ pub fn shr(lhs: Value, rhs: Value, ty: Type, ctx: Context) !Value {
 }
 
 pub fn compare(lhs: Value, op: std.math.CompareOperator, rhs: Value, ctx: Context) bool {
+    if (op == .eq) {
+        return lhs.opt_ref == rhs.opt_ref;
```

Comment:
I think we'll also need to check for "runtime" 1 & 0 vs "comptime" (statically known). e.g. the following will fail:

`_Static_assert(sizeof(char) == 1, "");` since the `Value.one` isn't the same as the computed value of `sizeof(char)`

Reply:
> since the `Value.one` isn't the same as the computed value of `sizeof(char)`

It is for me locally after I fixed a bug.